app/templates/agenda-items/agenda-item/index.hbs

```diff
@@ -235,6 +234,12 @@
                   @icon="decided"
                   @title="Dit werd behandeld"
                 />
+              {{else}}
```

Comment:
**Blocker:** Should be removed cf. https://github.com/lblod/frontend-burgernabije-besluitendatabank/pull/116/files#diff-c2141606cb920557e29cfaa8098d5b210e96f5756444fed2b47e7a216faf0244L218